packages/shorebird_cli/lib/src/code_push_client_wrapper.dart

```diff
@@ -656,6 +656,35 @@ aar artifact already exists, continuing...''',
     return thinnedArchiveDirectory;
   }
 
+  Future<void> createLinuxReleaseArtifacts({
+    required String appId,
+    required int releaseId,
+    required String bundlePath,
+  }) async {
+    final createArtifactProgress = logger.progress('Uploading artifacts');
+    final zippedBundle = await Directory(bundlePath).zipToTempFile();
+    try {
+      await codePushClient.createReleaseArtifact(
+        appId: appId,
+        releaseId: releaseId,
+        artifactPath: zippedBundle.path,
+        arch: 'bundle',
+        platform: ReleasePlatform.linux,
+        hash: sha256.convert(await zippedBundle.readAsBytes()).toString(),
+        canSideload: true,
+        podfileLockHash: null,
```

Comment:
Both of these are meaningless in this context, I wonder if they end up with defaults 🤷

Reply:
`canSideload` is needed for previewing-we use this to answer the question of whether the artifact can be run ~on its own (obviously we often need to unzip, so "on its own" is a bit murky)